src/nvim/optionstr.c

```diff
@@ -415,7 +415,7 @@ void set_string_option_direct_in_win(win_T *wp, const char *name, int opt_idx, c
 ///
 /// @return NULL on success, an untranslated error message on error.
 const char *set_string_option(const int opt_idx, const char *const value, const int opt_flags,
-                              char *const errbuf, const size_t errbuflen)
+                              int *value_checked, char *const errbuf, const size_t errbuflen)
```

Comment:
Maybe add a comment for the `value_checked` param. Also, could this be `bool *`? I only see it (and also `os_value_checked`) only be assigned to true/false